datasets/case02/queries/1002.2_coneMagColor.sql

```diff
@@ -0,0 +1,13 @@
+-- Cone-magnitude-color search
+-- See http://dev.lsstcorp.org/trac/wiki/dbQuery003
```

Comment:
Good it, it has been renamed to https://dev.lsstcorp.org/trac/wiki/db/queries/003